conda/core/prefix_data.py

```diff
@@ -165,7 +165,7 @@ def _prefix_records(self):
         return self.__prefix_records or self.load() or self.__prefix_records
 
     def _load_single_record(self, prefix_record_json_path):
-        log.trace("loading prefix record %s", prefix_record_json_path)
+        log.info("loading prefix record %s", prefix_record_json_path)
```

Comment:
The more I think about it, the more I think that `info` is the wrong loglevel. It's more likely that the right answer is either 1) `debug` or 2) patching `conda/core/prefix_data.py` so that it picks up the `trace` option properly.

Reply:
@kalefranz @mingwandroid thoughts? ^^^

Reply:
Thanks for the review, @mcg1969.

Reply:
@mrakitin of course! I'd love for @kalefranz to see this but he's out right for a bit. When he gets back I'll check in with him.

Reply:
Hello @mcg1969 and @kalefranz! Just encountered the same issue with programmatic access of the conda package list, and remembered I proposed a fix. It would be great to not celebrate an anniversary for this PR 😄. What would you recommend to unblock it? Thanks!